src/components/routineSingle.js

```diff
@@ -0,0 +1,27 @@
+const RoutineSingle = () => {
```

Comment:
- destructure the passed down prop in RoutineSingle
- example: 
const MyFunctionalComponent = ({ myProp }) => {
  // code here

  return (
    // JSX here
  )
}